package.json

```diff
@@ -36,14 +37,12 @@
   "devDependencies": {
     "@babel/eslint-parser": "7.17.0",
     "@babel/preset-react": "^7.16.7",
-    "@swc/core": "1.2.138",
-    "@swc/helpers": "0.3.2",
     "@types/fs-extra": "^9.0.13",
     "@types/wait-on": "5.3.1",
     "@typescript-eslint/eslint-plugin": "5.21.0",
     "@typescript-eslint/parser": "5.21.0",
     "algoliasearch": "4.12.1",
-    "chalk": "4.1.2",
+    "chalk": "5.0.1",
```

Comment:
Chalk v5 [is ESM-only](https://github.com/chalk/chalk/releases/tag/v5.0.0). I bumped this dependency to demo the new setup. The scripts would fail if they were still transpiled to CJS.